twake/backend/node/src/services/workspaces/services/workspace.ts

```diff
@@ -198,7 +198,7 @@ export class WorkspaceServiceImpl implements WorkspaceService {
     }/workspaces/${workspace.id}/thumbnail?t=${new Date().getTime()}`;
     let logoPublicUrl = undefined;
     if (workspace.logo) {
-      if (!item.logo || options.logo_b64) {
```

Comment:
Mmm could you check that it doesnt get triggered when we change only the name of the workspace for instance ?
All our APIs are made in a way we can send only partial update. So if we send partial update with only the name and not the logo, then the logo will be deleted.
A good alternative could be to add `(!item.logo && item.logo !== undefined)` Like this sending an empty string will remove the logo.

Reply:
item is usless here, item.logo is always filled here we cause we always get the workspace infos including `item.logo`.
When you press the delete btn to delete the logo item is not affected only options.logo_b64 is updated